API Blueprint Specification.md

```diff
@@ -478,6 +488,189 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none, [Action section](#def-action-section), [Request section](#def-request-section)
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header, list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
+
+**-- or --**
+
+Defined by the `Authentication` keyword in Markdown list entity followed by optional comma seperated list of scheme names which is prefixed by a colon.
+
+    + Authentication: <list>
+
+> **NOTE:** The second notation is only used when the parent section is either an [Action section](#def-action-section) or a [Request section](#def-request-section).
+
+#### Description
+Based on the parent section, the usage of this section differs as shown below.
+
+#### API Authentication section
```

Comment:
I would suggest to follow the [attributes approach](https://github.com/apiaryio/api-blueprint/blob/pksunkara/auth/API%20Blueprint%20Specification.md#description-16) – explain as much as you can in the general description – then mention where it can be used and then get into context-specific details.